rest/data/host_test.go

```diff
@@ -177,6 +177,9 @@ func (s *HostConnectorSuite) TestSpawnHost() {
 	testUser := &user.DBUser{
 		Id:     testUserID,
 		APIKey: testUserAPIKey,
+		Settings: user.UserSettings{
+			Timezone: "America/New_York",
```

Comment:
nit: evergreen.DefaultSleepScheduleTimeZone

Reply:
Hmm I thought about it, and this should probably test a different time zone because we want to verify that it'll prioritize the user's preference if they set it. Updated it to use a non-default time zone.